website/source/docs/templates/engine.html.md

```diff
@@ -22,9 +22,11 @@ need greater than one second granularity, you should use `{{uuid}}`, for
 example when you have multiple builders in the same template.
 
 In addition to globally available functions like `{{timestamp}}`, some
-configurations have special local variables that are available only for
-that configuration. These are recognizable because they're prefixed by a
-period, such as `{{.Name}}`.
+configurations have special template variables that are available only
```

Comment:
Here we have switched to using a specific phrase "template variables" but we have not defined it. Previously we said "local variables" which was sufficiently vague as to not need a definition.

However below we contrast "template variables" with "user variables" and it's clear that we never specified what "template variables" are, so we can't use them to define another term.

When I try to define "template variable" it strikes me that "local variables" actually makes more sense because they are "local" to the build. We could also possibly call them "build variables". In any case this is naming, which I don't care to bikeshed. What I do care about is, now that we have named them _something_, how we differentiate these variables from other types variables?

> "Template variables" are ____ .

- variables specific to this plugin?
- variables specific to the build that can be shared with provisioners and post processors?
- it varies?

In the example case `{{.Name}}` what does name refer to? The plugin name? the build name? etc. I think a general explanation like "these variables represent something specific to the build or plugin being invoked" and one or two examples should suffice here, but in the current form this is not particularly clear IMO.